src/Auth/Controller/AuthController.php

```diff
@@ -18,15 +18,14 @@
 
 final class AuthController
 {
-    private WebControllerService $webService;
     private ViewRenderer $viewRenderer;
-    private AuthService $authService;
 
-    public function __construct(ViewRenderer $viewRenderer, AuthService $authService, WebControllerService $webService)
-    {
+    public function __construct(
+        private AuthService $authService,
+        private WebControllerService $webService,
+        ViewRenderer $viewRenderer,
+    ) {
         $this->viewRenderer = $viewRenderer->withControllerName('auth');
```

Comment:
```suggestion
        $this->viewRenderer = $this->viewRenderer->withControllerName('auth');
```

Reply:
Both variants might be valid, need to check.

Reply:
It is OK as is.